### DISCUSSION TITLE
change in speed since 0.9.0

### DISCUSSION THREAD

**Discussion by martap79** [Python-Cardano/pycardano#258] [Q&A]
Hi All,

I've noticed a huge performance difference between pre-0.9.0 and 0.9.0. Building a TX now takes up to 2x as long with 0.9.0 as it did with pre-0.9.0. This is with type checking already disabled (also disabled in the 0.8.1 that I used to compare).

Any reason for this degradation in speed?

**Comment by nielstron** [comment]
Could you report what kind of tx you are building and what contexts you are using? Ideally showing the code and the measurements :)
